File: BackEnd/jeanstation/JeanStation.OrderService/Models/Order.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace JeanStation.OrderService.Models
{
    public class Order
    {
        [Key]
        public int OrderId { get; set; }
        public string OrderStatus { get; set; }
  
        public int UserId { get; set; }
      
        public int AddressId { get; set; }
        
        [MaxLength(1000)]
        public string OrderCreatedAt { get; set; }
        public int TotalProducts { get; set; }
        public int TotalPrice { get; set; }


    }
}

File: newuserui/src/app/admin/Model/Order.ts
export class Order {
    orderId: number;
    userId: number;
    addressId: number;
    totalProducts: number;
    totalPrice: number;
    //PaymentMethod:string;
    orderStatus:string;
    orderCreatedAt:string;
    

    constructor(orderid:number,userId:number,addressId:number,totalItems:number,totalOrderValue:number,orderStatus:string,orderPlacedAt:string){
        this.orderId = orderid;
        this.userId = userId;
        this.addressId =addressId;
        this.totalProducts =totalItems;
        this.totalPrice =totalOrderValue;
        //this.PaymentMethod = paymentMethod;
        this.orderStatus = orderStatus;
        this.orderCreatedAt = orderPlacedAt;
        
    }
}